internal/declcfg/diff.go

```diff
@@ -71,9 +94,11 @@ func Diff(oldModel, newModel model.Model) (model.Model, error) {
 		}
 	}
 
-	// Add dependencies to outputModel not already present in oldModel.
-	if err := addAllDependencies(newModel, oldModel, outputModel); err != nil {
-		return nil, err
+	if !g.NoDependencies {
```

Comment:
this is a nit, but I don't love double negatives. It would be nice if this was actually:

`if g.IncludeDependencies {`

Reply:
My general approach for struct fields (and especially for `bool`s) is to make their default value meaningful and the default that you actually want.

Since the default value for `bool`s is `false`, it would mean we want the `diff` functionality to default to including dependencies.

Reply:
I don't either, although I want the default to be false so dependencies are added by default. May just make this a pointer.

Reply:
Changed to `SkipDependencies`.